test_fms/drifters/test_drifters_comm.F90

```diff
@@ -59,6 +59,7 @@ program test_drifters_comm
 
   ! read input
   read (input_nml_file, drifters_comm_nml, iostat=io_status)
+  if(io_status > 0) call mpp_error(FATAL, 'test_drifters_comm: Error reading input.nml')
```

Comment:
Shouldn't we be using fms_mod::check_nml_error here?

Reply:
fixed in 7291341